lbz/websocket/request.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from multidict import CIMultiDict
+
+from lbz._request import _Request
+from lbz.authentication import User
+from lbz.misc import get_logger
+from lbz.websocket.enums import ActionType
+
+logger = get_logger(__name__)
+
+
+class WebSocketRequest(_Request):
+    """Represents request from Web Socket Secure API Gateway.
+
+    rel: https://docs.aws.amazon.com/apigateway/latest/developerguide/
+    apigateway-websocket-api-mapping-template-reference.html
+    """
+
+    def __init__(
+        self,
+        body: str | bytes | dict,
```

Comment:
Maybe it would be good to force using keyword arguments here?
```suggestion
        *,
        body: str | bytes | dict,
```

Reply:
Done - manually - had to check few things.